package.json

```diff
@@ -27,6 +27,7 @@
     "history": "^4.7.2",
     "localforage": "^1.5.5",
     "mini-css-extract-plugin": "^0.5.0",
+    "moment": "^2.24.0",
```

Comment:
Do we need this library?

Reply:
i have just deleted momentjs in next commit